### DISCUSSION TITLE
calendar and timeline plugin discussion

### DISCUSSION THREAD

**Discussion by TheBestOrNothing** [remarkjs/remark#739] [Ideas] [answered]
calendar and timeline plugin is very useful.  So do remark have plan to create these plugin? Thanks.

Calendar example:
![Calendar  example](https://simpleasthatblog.com/wp-content/uploads/2021/01/Screen-Shot-2021-01-04-at-5.55.00-PM.png)

 Timeline example:
![timeline](https://www.script-tutorials.com/wp-content/uploads/2016/06/result-700x487.png)

**Comment by Murderlon** [comment, marked as answer]
I personally don't see a way for markdown to have timelines and calendars. Although you could try to mimic a calendar with tables via [remark-gfm](https://github.com/remarkjs/remark-gfm). Also you might be able mimic a timeline somehow if you really tried with [remark-directive](https://github.com/remarkjs/remark-directive), compile to HTML, and creatively use CSS.

**Reply by wooorm** [reply]
Or maybe use a fenced code block with a flag of 'ics', and then paste a calendar file in there? 🤔
